src-java/testing/test-library/src/main/java/org/openkilda/testing/service/tsdb/model/StatsMetric.java

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2018 Telstra Open Source
+/* Copyright 2223 Telstra Open Source
```

Comment:
how's life there? OpenKilda is still in use as I see

Reply:
[Timeline](https://www.glow-internet.com/infographics/the-sci-fi-timeline/) is correct enough. After adding AI and moving to cloud, kilda evolved to what you, ancient people, called 'Matrix'. Digital copy of the so-named 'Kilda Architect' still tries to stabilize build, and agents are working (without success) on reducing amount of flaky tests :(